scripts/README.md

```diff
@@ -14,46 +14,83 @@ npx truffle compile
 yarn run networks-inject
 ```
 
-Setup the env variables:
+Create a gnosis-safe wallet [here](https://rinkeby.gnosis-safe.io). This wallet will be called your Master Safe in the following. It is used to bundle the transactions and setup the bracket-traders.
+This Master Safe must have an additional owner `0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1`, known as the "Proposer" account. The following scripts will use this account to propose transactions to the interface. This implies that the mnemonic phrase for this "Proposer" account is stored in plain text within this project.
+
+In order to have a save setup, make sure that _your Master Safe always requires one more signature than just the signature of the Proposer account to send a transaction_. _Otherwise, everyone can steal the funds from your account!_
+
+Setup env variables for the deployment process:
 
 ```
 export PK=<Your Key>
-export GAS_PRICE_GWEI=8
+export GAS_PRICE_GWEI=<look up the suggestion from ethgasstation.info>
 export NETWORK_NAME=<network>
 export MASTER_SAFE=<master safe>
 
 ```
 
-### Confirming multisig orders with Metamask
+### Confirming multisig-transactions on gnosis-safe with Metamask
+
+For the signing process, note that the gas consumption is underestimated. There is currently no proper way to estimate the gas limit. In order to make sure that the gas limit of your transaction is sufficient, increase it to 5 million gas. In practice, this limit suffices to deploy a strategy with 20 brackets.
+
+### Deploy the bracket-strategy:
 
-These script require to set up a "Master Safe" created through the interface (here)[https://rinkeby.gnosis-safe.io] with an owner key under your control. This Master Safe should have an additional owner `0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1`, known as the "Proposer" account. The following scripts will use this account to propose transactions to the interface and this implies that the mnemonic phrase for this "Proposer" account is stored in plain text within this project.
-In order to have a save setup, make sure that _your Master Safe always requires one more signature than just the signature of the Proposer account to send a transaction_. Otherwise, everyone can steal the funds from your account!
+In order to deploy new bracket-trader contracts, place orders on behalf of the newly mined contracts and fund their accounts on the exchange, one only has to run the `complete_liquidity_provision` script.
+It will create one ethereum transaction for creating the safes, and send two transactions request to the gnosis-safe interface.
```

Comment:
We could specify that the user's private key specified before is used to send just that one transaction (and has to pay for the gas) and the proposer account sends the other two transaction requests to the web interface (no ETH needed in the proposer account).